docs/tips-n-tricks/Wormholes/gas-harvesting/basics.md

```diff
@@ -42,7 +42,7 @@ Small Low Friction Nozzle Joints I
 ```
 
 Notes :
-- T1 gas scoops can be (and should be, for solo huffing) upgraded for T2 scoops
+- T1 gas scoops can be (and should be, for solo huffing) upgraded for T2 scoops. This requires Gas Harvesting V, and has a residue probability, so should not be used when huffing with a group.
```

Comment:
Depends on the size of the group and what you're trying to do. Better to say "Ask your FC when huffing with a group".